acmwebsite/public/css/style.css

```diff
@@ -1,32 +1,32 @@
 /* TurboGears flash bootstrap look */
 #flash > div {
-  padding: 8px 35px 8px 14px;
-  margin-bottom: 20px;
-  text-shadow: 0 1px 0 rgba(255, 255, 255, 0.5);
-  background-color: #fcf8e3;
-  border: 1px solid #fbeed5;
-  -webkit-border-radius: 4px;
-     -moz-border-radius: 4px;
-          border-radius: 4px;
-  color: #c09853;
+    padding: 8px 35px 8px 14px;
+    margin-bottom: 20px;
+    text-shadow: 0 1px 0 rgba(255, 255, 255, 0.5);
+    background-color: #fcf8e3;
+    border: 1px solid #fbeed5;
+    -webkit-border-radius: 4px;
+       -moz-border-radius: 4px;
+            border-radius: 4px;
+    color: #c09853;
```

Comment:
Indentation in CSS should use 2 spaces. Please don't change this.

Reply:
We've been very inconsistent with that throughout the CSS file so I was confused. I chose 4 because it seemed to be more common. (I actually prefer 2, so I'm happy to change this.)

Reply:
Added note to contributing guidelines.